### PR TITLE
Get Well: Add "Getting Started with Flex Gateway" doc

### PR DESCRIPTION
**Pull request by chris-campbell9**
# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [x] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [x] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 1]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+* A username and password for your Anypoint Platform organization. If you don’t have Anypoint Platform yet, https://anypoint.mulesoft.com/login/signup[create a Trial organization^] on Anypoint Platform.
+
+* A machine with an x86 processor. M1 and M2 processors are not supported.
+
+* A Docker installation. See https://www.docker.com/products/docker-desktop/[Docker Desktop^] for an example.
+
+== Install Flex Gateway
+
+To install the Flex Gateway Docker image, run the following command in a terminal:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
To install the Flex Gateway Docker image, run the following command in a terminal window:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 2]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+----
+docker pull mulesoft/flex-gateway:latest
+----
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+== Register Flex Gateway
+
+. Create a new directory by running the following command:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
. Create a new directory:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 3]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+[source,ssh]
+----
+mkdir flex-config
+----
+
+. Navigate to the new directory by running the following command:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
. Navigate to the new directory:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 4]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+----
+cd flex-config
+----
+
+. Log in to https://anypoint.mulesoft.com/login/[Anypoint Platform^] with your Anypoint username and password.
+. Select *Runtime Manager*.
+. Click on the *Flex Gateways* tab in the side navigation panel.
+. Click *Add Gateway*.
+. Select *Docker*.
+. Copy the code block in the *Register your gateway* section. The code block will contain your unique organization ID and token.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
. Copy the code block in the *Register your gateway* section. The code block contains your unique organization ID and token.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 5]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+The following screenshot of the *Add a Flex Gateway* page highlights the code block that should be copied.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
The following screenshot of the *Add a Flex Gateway* page highlights the code block to copy:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 6]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+image::gateway-quickstart-add-gateway.png["Screenshot of 'Add a Flex Gateway'",width=90%]
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
image::gateway-quickstart-add-gateway.png["Add a Flex Gateway page with selected code block to copy",width=90%]
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I believe we don't want to use "screenshot" in the alt text.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 7]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+. Select *Flex Gateways* in the side navigation panel.
++
+Your registered Flex Gateway now appears in the list of gateways.
+
+== Run Flex Gateway
+
+To run Flex Gateway, run the following command:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
To run Flex Gateway, enter the following in a terminal window:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 8]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+A new Flex Gateway instance is now running in your terminal window.
+
+== Publish a Simple API
+
+. Select *API Manager* from the Anypoint Platform menu.
+. Click *Add API*, followed by *Add new API*.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
. Click *Add API*, followed by *Add new API*:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 9]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+image::gateway-quickstart-add-new-api.png["Screenshot of 'Add new API'",width=35%]
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: see prev about alt text

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 10]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+. In the *Add API* page, select *Flex Gateway* as your runtime.
+. Select your registered gateway in the *Select a gateway* section.
+. Click *Next*.
++
+The following screenshot of the *Add API* page highlights the specified options.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
The following screenshot of the *Add API* page highlights the specified options:
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Kim's guidance was always introduce an image with a colon (:)

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 11]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+image::gateway-quickstart-select-gateway-for-api.png["Screenshot of 'Select Gateway for API'",width=100%]
+
+. Click *Create New API*.
+. Specify an API name and select *HTTP API* as the asset type.
+. Click *Next*.
+. In the *Endpoint* page, enter the following for *Implementation URI*:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: is this a field? Maybe enter xx in the xx field?

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 12]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+[source]
+----
+https://jsonplaceholder.typicode.com/
+----
+. For *Port*, enter `8081`.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: same as 102

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 13]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+. Click *Next*.
++
+// The following screenshot of the *Endpoint* page highlights the specified options.
+// +
+// image::gateway-quickstart-select-endpoint.png["Screenshot of 'Endpoint' page",width=100%]
+
+. Click *Save & Deploy*.
++
+Inside the gateway instance Docker container, all HTTP requests made to `localhost` on port `8081` will now be proxied to the `jsonplaceholder` service.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
Inside the gateway instance Docker container, all HTTP requests made to `localhost` on port `8081` are now proxied to the `jsonplaceholder` service.
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I assume it's ok to use proxy as a verb :>

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 14]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+. To test the API Instance, run the following command in a new terminal window:
++
+[source,ssh]
+----
+curl -s -o /dev/null -w "%{http_code}" --request GET 'http://localhost:8081/'
+----
++
+The command executes a `GET` request to the API Instance, and subsequently prints the resulting `200` status code, indicating success.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
The command executes a `GET` request to the API instance, and then prints the resulting `200` status code, indicating success.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 15]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+== Secure Your API Using the Basic Authentication Policy
+
+. While still in *API Manager*, select *API Administration* in the side navigation panel.
+. Select the highlighted name of the API Instance created in the previous section.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: hmmm. Maybe Instance is a proper noun?

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 16]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+[source,ssh]
+----
+curl -s -o /dev/null -w "%{http_code}" --request GET 'http://localhost:8081/'
+----
++
+The command executes a `GET` request to the API Instance, and subsequently prints the resulting `200` status code, indicating success.
+
+== Secure Your API Using the Basic Authentication Policy
+
+. While still in *API Manager*, select *API Administration* in the side navigation panel.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Need to check the Style Ref about select "in" or "from"

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 17]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+. Select the highlighted name of the API Instance created in the previous section.
+. Select *Policies* in the side navigation panel.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: from?

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 18]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+. In the *Policies* page, click *Add policy*.
++
+The following screenshot of the *Policies* page highlights the specified options.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
The following screenshot of the *Policies* page highlights the specified options:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 19]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+image::gateway-quickstart-add-policy.png["Screenshot of 'Add Policy'",width=65%]
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: alt text

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I don't think we ever use single quotes.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 20]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+. Select the *Basic Authentication - Simple* policy.
+. Click *Next*.
+. For *User Name*, enter `user`.
+. For *User Password*, enter `password`.
+. Click *Apply*.
++
+The following screenshot of the *Configure Basic Authentication - Simple policy* page highlights the specified options.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]:
```suggestion
The following screenshot of the *Configure Basic Authentication - Simple policy* page highlights the specified options:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 21]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
++
+image::gateway-quickstart-config-basic-auth.png["Screenshot of 'Configure Basic Auth Policy'",width=65%]
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: alt

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc [hunk 22]
```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
...
+
+. To test the API Instance _without_ authentication, run the previous `curl` command:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: previous?

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 1 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 2 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 3 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 4 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 5 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 6 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 7 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Hmm, I think I see what you're getting at. But they're not just entering text; they're also running the command.

Maybe:
"To run Flex Gateway, execute the following in a terminal window"?

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 8 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 10 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 11 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 12 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 13 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 14 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed. I removed "instance", as it was redundant.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 15 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I removed "instance", as it was redundant.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 18 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 20 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 22 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed. I replaced "previous" with "following".

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 9 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 19 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 21 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: Fixed

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 16 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I checked the guide. It should be "from". I fixed this in a couple of different places.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 17 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: I checked the guide. It should be "from". I fixed this in a couple of different places.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc (hunk 7 above)
Comment (chris-campbell9) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc]: "Run" it is! :)

**Review by hannanelson — APPROVED** [review]

**Pull request merged**